### ISSUE TITLE
Emerging Hot Spot Analysis

### ISSUE THREAD

**Issue by martinfleis** [pysal/esda#221]
See this from the R side https://sfdep.josiahparry.com/articles/understanding-emerging-hotspots.html. It would be cool to have a similar functionality here as well.

**Comment by ljwolf** [comment]
This looks like some of the techniques in giddy, like the LISA_Markov?

**Comment by martinfleis** [comment]
Yeah, that seems close. The `move_type` can be interpreted as their `consecutive hotspot`, `oscilating hotspot` etc. Shall we close this or move over to giddy?

**Comment by ljwolf** [comment]
> move type
Precisely. 

> close and move to giddy
I think it might be useful to keep this open to add an example notebook to esda saying how to use Lisa_Markov? Esda (the package) and giddy are kind of in the same "Task view" for ESDA (the practice)...
